daemon/exercise.go

```diff
@@ -45,11 +47,16 @@ func (d *daemon) ListCategories(ctx context.Context, req *pb.Empty) (*pb.ListCat
 
 	for _, c := range cats {
 		// Render markdown from orgdescription to html
-		extensions := parser.CommonExtensions | parser.AutoHeadingIDs | parser.HardLineBreak
-		parser := parser.NewWithExtensions(extensions)
 
 		md := []byte(c.CatDescription)
-		unsafeHtml := markdown.ToHTML(md, parser, nil)
+		var buf bytes.Buffer
+		renderer := goldmark.New(
+			goldmark.WithRendererOptions(html.WithUnsafe()),
+		)
+		if err := renderer.Convert(md, &buf); err != nil {
+			log.Error().Msgf("Error converting to commonmark: %s", err)
```

Comment:
```suggestion
			log.Error().Err(err).Msg("converting to markdown")
```
Use the fields of the log package, might be nice if decided to do central logging for the distributed version later on as it makes it nicer to filter logs